components/layout/Nav.js

```diff
@@ -30,12 +33,21 @@ const Nav = () => {
               )}
             </button>
           )}
-        </>
+        </section>
       ) : (
-        <>
-          <Link href="/log-in">Log In</Link>
-          <Link href="/sign-up">Sign Up</Link>
-        </>
+        <section>
```

Comment:
As the `StyledNav` is just the container for multiple lists of links, I would say that this `section` is a  `nav`